x/ccv/consumer/keeper/relay.go

```diff
@@ -155,8 +157,10 @@ func (k Keeper) SendSlashPacket(ctx sdk.Context, validator abci.Validator, valse
 		panic(err)
```

Comment:
Can it be replaced with sendIBCPacket util introduced in https://github.com/cosmos/interchain-security/pull/86?

Reply:
Yea sure. Thanks for pointing this out.